components/hab/src/command/pkg/binlink.rs

```diff
@@ -106,6 +118,22 @@ where
             if bin_file.file_type()?.is_dir() {
                 continue;
             }
+            if cfg!(target_os = "windows") {
+                match bin_file.path().extension() {
+                    Some(ext) => match env::var_os("PATHEXT") {
+                        Some(val) => {
+                            if !env::split_paths(&val).any(|e| {
+                                e.to_string_lossy()
```

Comment:
Is it a guarantee that the values from `PATHEXT` will be uppercase?

Reply:
that is how windows will render it but there is no stopping someone from changing it. I'll force it to uppercase just in case.